Sources/StreamChat/Query/ChannelListQuery.swift

```diff
@@ -42,9 +42,12 @@ extension Filter where Scope: AnyChannelListFilterScope {
 }
 
 // We don't want to expose `members` publicly because it can't be used with any other operator
-// then `$in`. We expose it publicly via the `containMembers` filter helper.
+// than `$in`. We expose it publicly via the `containMembers` filter helper.
+// TODO: should it be using the ChannelPayload.members or ChannelPayload.channel.members
 extension FilterKey where Scope: AnyChannelListFilterScope {
-    static var members: FilterKey<Scope, UserId> { "members" }
+    static var members: FilterKey<Scope, UserId> { .init(rawValue: "members") {
+        ($0 as? ChannelPayload)?.members.map(\.userId)
```

Comment:
Is this the right keypath for the members or shall we instead get from the `($0 as? ChannelPayload)?.channel.members.map(\.userId)`?